packages/big-design/src/components/Worksheet/Cell/Cell.tsx

```diff
@@ -248,6 +257,26 @@ const InternalCell = <T extends WorksheetItem>({
     renderedValue,
   ]);
 
+  const renderedNote = useMemo(() => {
+    if (!notation) {
+      return null;
+    }
+
+    const note = notation(value, row);
```

Comment:
Should we also check if note is `undefined` and return `null`? 🍹 

```tsx
 if (!note) {
   return null
 }

return (
    <Tooltip>
        ....
    </Tooltip>
)
```

Reply:
I think it will not be redundant to avoid returning undefined. So I`ve updated the code according to your suggestion.